.ci/docker/conda-env-ci.txt

```diff
@@ -1,4 +1,5 @@
 cmake=3.22.1
 ninja=1.10.2
 libuv
+llvm-openmp
```

Comment:
Is this one needed given that there is this line to install libomp here https://github.com/pytorch/executorch/blob/main/.ci/docker/common/install_clang.sh#L16?

Reply:
IIRC @malfet told me to add this one